public/docs/js/latest/cookbook/ts-to-js.jade

```diff
@@ -24,90 +26,127 @@ include ../../../../_includes/_util-fns
-        from `@angular` in TypeScript:
-
-      +makeExample('cb-ts-to-js/js/app/main.js', 'ng2import')(format="." )
-
-  tr(style=top)
-    td
-      :marked
-        ### Importing and Exporting Application Code
-
-        Each file in an Angular TypeScript application constitutes a
-        TypeScript module. When we want to make something from a module available
-        to other modules, we `export` it.
-
-      +makeExample('cb-ts-to-js/ts/app/hero.component.ts', 'appexport')(format="." )
-
-      :marked
-        In other modules we can then `import` things that have been exported
-        elsewhere.
-
-      +makeExample('cb-ts-to-js/ts/app/main.ts', 'appimport')(format="." )
+  ### Importing Angular Code
+
+  In TypeScript and ES6 Javascript, Angular classes, functions, and other members are imported
+  with ES6 `import` statements.
+
+  In ES5 JavaScript code, when using [the Angular packages](../glossary.html#!#scoped-package),
+  we can access Angular code through the global `ng` object. In the nested members of this 
+  object we'll find everything we would import from `@angular` in TypeScript:
+
++makeTabs(`
+    cb-ts-to-js/ts/app/main.ts,
+    cb-ts-to-js/js-es6-decorators/app/main.es6,
+    cb-ts-to-js/js-es6/app/main.es6,
+    cb-ts-to-js/js/app/main.js
+  `,`
+    ng2import,
+    ng2import,
+    ng2import,
+    ng2import
+  `,`
+    Typescript,
+    ES6 Javascript with decorators,
+    ES6 Javascript,
+    ES5 Javascript
+  `)
 
-    td
-      :marked
-        ### Sharing Application Code
-
-        In an Angular JavaScript application, we load each file to the page
-        using a `<script>` tag. Each file can make things available to other
-        files via the shared global `window` scope.
-
-        We often introduce an application namespace
-        object (such as `"app"`) onto `window` and attach everything we need
-        to share to that namespace object. 
-        We also wrap our code in an 
-        [Immediately Invoked Function Expression (IIFE)](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression).
-        These practices together prevent our code from
-        polluting the global scope.
-
-      +makeExample('cb-ts-to-js/js/app/hero.component.js', 'appexport')(format="." )
-
-      :marked
-        We can then access anything from this shared namespace in
-        other files.
-
-      +makeExample('cb-ts-to-js/js/app/main.js', 'appimport')(format="." )
+:marked
+  ### Importing and Exporting Application Code
+
+  Each file in an Angular TypeScript or ES6 Javascript application constitutes a ES6 module.
+  When we want to make something from a module available to other modules, we `export` it.
+
+  In an Angular ES5 JavaScript application, we load each file to the page using a `<script>` tag. 
+  Each file can make things available to other files via the shared global `window` scope.
+
+  We often introduce an application namespace object (such as `"app"`) onto `window` and attach
```

Comment:
Do we really need double quotes in \`"app"\`? It is a variable, the same as \`window\`, and that is mentioned without double quotes.